frontend/fingertips-frontend/components/pages/results/index.tsx

```diff
@@ -40,7 +46,7 @@ export function SearchResults({
   searchResults,
   availableAreaTypes,
 }: Readonly<SearchResultsProps>) {
-  const [state, formAction] = useActionState(
+  const [viewChartsState, viewChartsFormAction] = useActionState(
```

Comment:
I can see why you have named it like this. But this can be little misleading with the charts pages. In hindsight naming the function viewCharts probably wasn't the best idea.

I think we need to make it clear this refers to indicatorSelection state and form action.

Can we rename to `indicatorSelectionState` and `indicatorSelectionFormAction`

rename the `viewCharts` function to `submitIndicatorSelection`